frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -36,17 +36,23 @@ export interface OperatorValueSelectProps {
     operatorSelectProps?: Partial<Omit<LemonSelectProps<any>, 'onChange'>>
     eventNames?: string[]
     propertyDefinitions: PropertyDefinition[]
-    defaultOpen?: boolean
     addRelativeDateTimeOptions?: boolean
     groupTypeIndex?: GroupTypeIndex
     size?: 'xsmall' | 'small' | 'medium'
+    startVisible?: LemonDropdownProps['startVisible']
```

Comment:
startVisible does work and doesn't confuse me :)
especially since the thing it controls is `visible` and not `open`